test/CDB_CartodbfyTableTest.sql

```diff
@@ -112,15 +102,6 @@ BEGIN
       RAISE EXCEPTION '% gist indices found on the_geom and the_geom_webmercator, expected 2', tmp;
   END IF;
 
-  -- Check null constraint on cartodb_id, created_at, updated_at
-  SELECT count(*) FROM pg_attribute a, pg_class c WHERE c.oid = tabname::oid
-    AND a.attrelid = c.oid AND NOT a.attisdropped AND a.attname in
-      ( 'cartodb_id', 'created_at', 'updated_at' )
-    AND NOT a.attnotnull INTO strict tmp;
-  IF tmp > 0 THEN
-      RAISE EXCEPTION 'cartodb_id or created_at or updated_at are missing not-null constraint';
-  END IF;
-
```

Comment:
You've lost the not-NULL test on cartodb_id here, not sure if this table test function is used anywhere though.